### ISSUE TITLE
Default value for "Version"

### ISSUE THREAD

**Issue by pxwee5** [storyblok/storyblok#13]
**The issue occurs while working with:** (check one with "x")   
- [x] *api.storyblok.com*   

**I'm submitting a ...**  (check one with "x")  
- [x] bug report 

**Current behavior:**
Default value not clearly stated in the docs

**Expected behavior:**
Should state the default version that is pulled if version is not set. 

How does the `published` or `draft` version differ from cv.

**Comment by DominikAngerer** [comment]
Hey there.

The Published / Draft indicates the Version status of the Content entry - this can be changed directly in the interface and decided by the user itself.

The `cv` parameter is used to allow your requests to be cached for a longer period in our CDN - to allow you to access that content as fast as possible. As Storyblok uses a CDN in front of the API to deliver the response in the fastest way possible, you need to append the cv parameter to the API, the Cache Version can be found by doing an initial request on [Spaces](https://www.storyblok.com/docs/Delivery-Api/spaces) the `space.version` parameter in this case would be a timestamp of the last modified state and only changes if one entry of your content changes. 

So in short:

`cv` - CDN caching timestamp or parameter to allow longer caching periods
`version` - `published` or `draft` to switch between Draft or Published Content Version

**Comment by pxwee5** [comment]
And if I'm running on a Nuxt app, my store should obtain the `cv` from `cdn/spaces/me` when the app starts. And all future requests should all refer to the `space.version` property correct? 

Are there any instances where I would not use cv in my requests?

**Comment by DominikAngerer** [comment]
That is correct. You can either use the version you receive from that `cdn/spaces/me` call or define your own Versioning strategie - our [JS Client](https://github.com/storyblok/storyblok-js-client/blob/master/source/index.js) for example uses a self generated timestamp. 

Some plugins/custom field types rely on "current" data - and therefore they don't want to use caching - so they would use `cv` with a new timestamp instead of creating new requests. Also for static site generator the `cv` can simply be a timestamp since it would be built once. :) No need for that extra request.
